src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -218,37 +228,38 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
 
     const depositInput = await getDepositInput()
     if (!depositInput) throw new Error('Error building send input')
-
-    let txId: string
+    if (!walletState?.wallet) throw new Error('Wallet is required')
 
     // Try/catching and evaluating to something in the catch isn't a good pattern usually
     // In our case, handleSend() catching means that after all our previous checks, building a Tx failed at coinselect time
     // So we actually send reconciliate a reconciliate Tx, retry the original send within the same block
     // and finally evaluate to either the original Tx or a falsy empty string
-    try {
-      // 1. Try to deposit from the originally deposited from / highest UTXO balance address
-      // If this is enough, no other Tx is needed
-      txId = await handleSend({
-        sendInput: depositInput,
-        wallet: walletState.wallet,
-      })
-    } catch (e) {
+    // 1. Try to deposit from the originally deposited from / highest UTXO balance address
+    // If this is enough, no other Tx is needed
+    const txId = await handleSend({
+      sendInput: depositInput,
+      wallet: walletState.wallet,
+    }).catch(async e => {
+      if (!isUtxoChainId(chainId)) throw e
```

Comment:
If this fails, this is an actual sign/broadcast error, not an UTXO coinselect one no need to intent a pre-withdraw Tx